### PR TITLE
Fix profiler job

### PR DESCRIPTION
**Pull request by pierDipi**
As per title, flamegraphs can be downloaded from
https://github.com/knative-sandbox/eventing-kafka-broker/actions/runs/2753216718

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: data-plane/profiler/run.sh [hunk 1]
```diff
@@ -101,7 +101,7 @@ export WAIT_STARTUP_SECONDS="8"
 export SERVICE_NAME="kafka-broker-receiver"
 export SERVICE_NAMESPACE="knative-eventing"
 export INGRESS_PORT="8080"
-export METRICS_PORT="9090"
+export METRICS_PORT="9098"
 export INSTANCE_ID="receiver"
```
Comment (aavarghese) [inline data-plane/profiler/run.sh:105]: Why the port # change?

Comment (pierDipi) [reply data-plane/profiler/run.sh:105]: Because sacura started to expose metrics on 9090, so that broke this job :)

Comment (matzew) [reply data-plane/profiler/run.sh:105]: :thinking:

Comment (pierDipi) [reply data-plane/profiler/run.sh:105]: I started to expose metrics on 9090 for perf testing on sacura side, so the receiver and sacura are trying to use the same local port 

https://github.com/pierDipi/sacura/commit/8d35ca8ae2beb65c7e5764d5d25efa0a018f30b4

**Comment by aavarghese** [comment]
/test upgrade-tests_eventing-kafka-broker_main
--


<br class="Apple-interchange-newline">

**Review by aavarghese — APPROVED** [review]
/lgtm

**Pull request merged**